doc/cookbook/engine.rst

```diff
@@ -0,0 +1,84 @@
+Engines
+=======
+
+Ploomber currently supports 2 engines for running notebooks:
+
+- Ploomber Engine
+- Papermill Engine
+
+
+`Ploomber Engine <https://engine.ploomber.io/en/latest/quick-start.html>`_
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Ploomber Engine is an in-house engine developed on top of Papermill, 
+with better support for error debugging and pipeline deployment. 
+Developers can use Ploomber Engine with Ploomber by setting the ``ploomber_engine`` argument to ``True`` 
+in ``NotebookRunner``. Additionally, developers can pass the following `arguments <https://engine.ploomber.io/en/latest/api/api.html#execute-notebook>`_ in ``nb_executor_params`` while using ``NotebookRunner`` with ``ploomber-engine``.
+
+
+`Papermill <https://papermill.readthedocs.io/en/latest/>`_
```

Comment:
I'm confused about this part:

```
By default, if the ``executor`` argument is not specified in 
``NotebookRunner``, and ``ploomber_engine`` is set to ``False``, 
Ploomber will use Papermill to execute notebooks.
```

does this mean we have *two* options for selecting the executor/engine? we should only have one